src/excmds.ts

```diff
@@ -692,6 +692,8 @@ export async function nativeopen(...args: string[]) {
  *
  * Requires the native messenger, obviously.
  *
+ * If you're using `exclaim` with arguments coming from a pipe, consider using `shellescape` to properly escape arguments and to prevent unsafe commands.
```

Comment:
```suggestion
 * If you're using `exclaim` with arguments coming from a pipe, consider using [[shellescape]] to properly escape arguments and to prevent unsafe commands.
```